packages/web-client/app/utils/web3-strategies/layer-1-connection-manager.ts

```diff
@@ -149,8 +149,10 @@ export class ConnectionManager {
 
   async connect(web3: Web3, providerId: WalletProviderId) {
     await this.setup(providerId);
+    if (!this.strategy)
+      throw new Error('Failed to setup strategy in layer 1 connection manager');
     web3.setProvider(this.provider);
-    await this.strategy?.connect();
+    return await this.strategy.connect();
```

Comment:
This change was to make this function consistently return a boolean if setup works correctly.